packages/office-ui-fabric-react/src/components/Tooltip/examples/Tooltip.Basic.Example.tsx

```diff
@@ -8,8 +8,8 @@ export class TooltipBasicExample extends BaseComponent<any, any> {
   public render(): JSX.Element {
     return (
       <div>
-        <TooltipHost content="This is the tooltip" id="myID" calloutProps={{ gapSpace: 0 }}>
-          <DefaultButton aria-describedby="myID">Hover Over Me</DefaultButton>
+        <TooltipHost content="This is the tooltip" id="myID-base" calloutProps={{ gapSpace: 0 }}>
```

Comment:
Might be good to do a search for ` id=` in the tooltip examples folder and make sure they're all de-duplicated...I've noticed this problem across a lot of examples for different components. :( 

Also, might be better to go with IDs that are more specific to the particular example--in this case, something like `id="basicHost"` maybe.

Oh or even better, create a private member `_hostId` and generate it using `getId()` in the constructor (or using inline property initialization)? Guaranteed uniqueness even if someone copy-pastes.

Reply:
i really don't like extra functions people have to digest here.... perhaps the uniqueness of the ID problem can be solved if examples were inside iframes (yuck but maybe we can embed codepen or codesandbox)